pkg/reduce/data_forward_test.go

```diff
@@ -348,7 +348,7 @@ func TestDataForward_StartWithNoOpWM(t *testing.T) {
 
 // ReadMessage size = 0
 func TestReduceDataForward_IdleWM(t *testing.T) {
-	//FIXME: fix this test
+	// FIXME: fix this test
```

Comment:
@yhl25 Do you remember what the bug is?... I ran this locally but didn't get any error..